examples/common/example_scope_time.cpp

```diff
@@ -48,11 +48,7 @@ main ()
 {
   pcl::ScopeTime scope_time ("Test loop");
   {
-    float total = 0.0f;
```

Comment:
This is example code and might be serving the purpose to demonstrate scope_time. Using the total for cout might be better in this case since the compiler will optimize out any loop without side-effects